src/main/scala/nl/knaw/dans/easy/archivebag/EasyArchiveBag.scala

```diff
@@ -85,21 +90,64 @@ object EasyArchiveBag {
     }
     val deleted = tempFile.delete()
     if(!deleted) log.warn(s"Delete of $tempFile failed")
-    response.getStatusLine.getStatusCode match {
+    val seIri = response.getStatusLine.getStatusCode match {
       case 201 =>
         val location = response.getFirstHeader("Location").getValue
         log.info("SUCCESS")
         log.info(s"Deposit created at: $location")
         s"${location.split('/').last}/${s.bagDir.getName}"
+
       case _ =>
         val writer = new StringWriter()
         IOUtils.copy(response.getEntity.getContent, writer, "UTF-8")
         log.error(s"Deposit failed: ${response.getStatusLine}, ${writer.toString}")
         throw new RuntimeException("Deposit failed")
     }
+    log.info(seIri)
```

Comment:
When logging keep in mind:
- Make the log statements informative. How are we supposed to know what the IRI means when we see above statement appear in the log (I mean, without inspecting this code)?
- Choose the `info` level only if appropriate. `info` is what you want the application manager to see. Here `debug` seems more to the point.